compose-settings-ui-m3/src/main/kotlin/com/alorma/compose/settings/ui/SettingsListMultiSelect.kt

```diff
@@ -45,6 +47,7 @@ fun SettingsListMultiSelect(
   }
 
   var showDialog by remember { mutableStateOf(false) }
+  val scrollState = rememberScrollState()
```

Comment:
Maybe scrollState must be a param, as in other composables, like lazy column. So it can be used from outside.

What do you think?

Reply:
Since this is a self-contained dialog, probably not. It'd only make sense to move that up the chain if the for loop also got moved up.